CHANGES.md

```diff
@@ -1,5 +1,26 @@
 # Changelog
 
+## Dendrite 0.9.7 (2022-09-09)
+
+### Features
+
+* Initial supporting code to enable full-text search has been merged (although not ready for use yet)
+* Newly created rooms now have higher default power levels for enabling encryption, setting server ACLs or sending tombstone events
```

Comment:
xref spec?

Reply:
I don't think it's in the spec, but these three things probably shouldn't be PL 50 and Synapse sets them to PL 100 as default too.